comps/comps-foreman-rhel7.xml

```diff
@@ -166,6 +166,7 @@
       <packagereq type="default">tfm-rubygem-statsd-instrument</packagereq>
       <packagereq type="default">tfm-rubygem-syntax</packagereq>
       <packagereq type="default">tfm-rubygem-text</packagereq>
+      <packagereq type="default">tfm-rubygem-typhoeus</packagereq>
```

Comment:
Should these be added to katello, not foreman comps?

Reply:
fixed, appears like the add package script doesn't handle this correctly